### ISSUE TITLE
[BUG] Bloxstrap bluescreening

### ISSUE THREAD

**Issue by DS4RV** [bloxstraplabs/bloxstrap#2994]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [ ] I did not answer truthfully to all the above checkboxes.

### What problem did you encounter?

The first 3 days bloxstrap worked fine but after that when I opened bloxstrap it kept bluescreening me. I tried deleting all the files and deleting Roblox as well but when I downloaded bloxstrap and opened the file it blue screened again. I can’t even install it now, what should I do?

### Bloxstrap Log

N/A

**Comment by pizzaboxer** [comment]
What error code do you get? Have you tried reinstalling the .NET 6 runtime? https://aka.ms/dotnet-core-applaunch?missing_runtime=true&arch=x64&rid=win11-x64&apphost_version=6.0.16&gui=true

**Comment by MrT-tak** [comment]
> ### Acknowledgement of preliminary instructions
> * [x]  I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
> * [x]  I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
> * [x]  I am using the latest version of Bloxstrap.
> * [ ]  I did not answer truthfully to all the above checkboxes.
> 
> ### What problem did you encounter?
> The first 3 days bloxstrap worked fine but after that when I opened bloxstrap it kept bluescreening me. I tried deleting all the files and deleting Roblox as well but when I downloaded bloxstrap and opened the file it blue screened again. I can’t even install it now, what should I do?
> 
> ### Bloxstrap Log
> N/A

I dont really want to give technical advice since im not too knowledgeable about c# and windows but have you seen the error codes and/or the log in event viewer and see what is causing it? 
Bluescreen crashes are usually because of a issue with your driver i think usually because ever since windows moved to NT technology, most apps when they error out, it doesnt cause a BSOD and just quit without disturbing other stuff. The crowdstrike world wide blue screen event happened because the driver for the security system was broken and it was set to required to use Windows, therefore Windows blue screened to protect files and prevent stuff from breaking.

TLDR Usually somekind of driver is included in the troubles. Check your drivers and find error codes (event viewer) and look it up please.

**Comment by pizzaboxer** [comment]
Issue has stale for over a month, and will be reopened if further information is given.
